auth-web/src/components/auth/manage-business/AffiliationAction.vue

```diff
@@ -280,7 +265,17 @@ export default defineComponent({
     }
 
     /** Emit business/nr information to be unaffiliated. */
-    const removeBusiness = (business: Business): void => {
+    const removeBusiness = async (business: Business): Promise<void> => {
```

Comment:
Should remove affiliation invitations as well provided they're in PENDING, FAILED, EXPIRED.